test/Graphics/Shape.cpp

```diff
@@ -1,4 +1,6 @@
+#include <SFML/Graphics/Rect.hpp>
 #include <SFML/Graphics/Shape.hpp>
+#include <SFML/System/Vector2.hpp>
```

Comment:
Why did you add these headers? I don't believe they're necessary and make it harder to find bugs related to missing headers in `Shape.hpp`.

Reply:
Indirect inclusion?

Reply:
Please remove these headers. Adding these headers is unnecessary and outside of the scope of this PR.